src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/auth/AuthPageContext.java

```diff
@@ -0,0 +1,54 @@
+package com.wikia.webdriver.pageobjectsfactory.pageobject.auth;
+import com.wikia.webdriver.common.logging.PageObjectLogging;
+import com.wikia.webdriver.pageobjectsfactory.componentobject.modalwindows.FacebookSignupModalComponentObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.AttachedRegisterPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.register.RegisterPage;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.auth.signin.AttachedSignInPage;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+public class AuthPageContext extends WikiBasePageObject implements FacebookAuthContext {
+
+  @FindBy(css = ".signup-provider-facebook")
+  private WebElement facebookSignUpButton;
+  @FindBy(css = ".signup-providers li a")
```

Comment:
they were actually pointing to the same DOM reference, thanks for helping me spot that